src/ResourceManager/AnalysisServices/ChangeLog.md

```diff
@@ -21,6 +21,7 @@
 
 ## Version 0.6.7
 * Set minimum dependency of module to PowerShell 5.0
+* Enable Gateway assocaite/disassociate operations on AS.
```

Comment:
This should go directly under Current Release... We will move it to under a new version header when we generate the release version.